index.bs

```diff
@@ -44,6 +44,105 @@ url: https://dom.spec.whatwg.org/#ref-for-concept-getelementsbytagname; type: in
     }
 </style>
 
+<h2 id="api-principles">Principles behind design of Web APIs</h2>
+
+<h3 id="safe-to-browse">It should be safe to visit a web page</h3>
+
+Hyperlinks, links from one page to another, are one of the foundations of the Web.
+
+Following a link, or visiting a web page, should be safe.
+Users who are following a link to a web page they haven't seen before
+should not have to fear for the security of their computer
+or for essential aspects of their privacy.
+(One caveat here is that users of the Web
+should not assume that they are free from consequences
+if their use of the Web is harming others.)
+
+Saying “essential aspects” here skips over quite a bit of detail.
```

Comment:
I'm also a little unclear on what this paragraph is saying. What user interfaces are you referring to? Can we rephrase this to focus on the point about user expectations and privacy, and either omit the aspirational aspects or give them their own paragraph?

Reply:
I've tried to do this; I think I pulled the aspirational bits out into their own paragraph, which I actually ended up structuring as a list.